src/pages/Transaction.vue

```diff
@@ -66,6 +73,24 @@ export default {
             },
             immediate: true,
         },
+        $route: {
```

Comment:
this is identical to the implementation in `Staking.vue`, refactor if practical.